### ISSUE TITLE
cannot start nodester app

### ISSUE THREAD

**Issue by piatra** [nodester/nodester#236]
nodester info { response: '{"status":"failure","message":"No logs available."}\n',
  errorCode: 500 }
nodester ERROR Fatal Error! API Response: {"status":"failure","message":"No logs available."}

nodester info Attemping to start app: twtups
nodester warn false

I can access the app but i cannot start it.Also how do i push updates via git ?

**Comment by sheldon** [comment]
I'm pretty sure nodester.com is having some troubles since their new keys were given out on friday. I've been in touch with @chrismatthieu on irc & twitter and he's aware of the problem, I'm sure it'll be fixed as soon as they have time to look into it.

**Comment by yocontra** [comment]
Looking into this now, sorry for the inconvenience. Hoping to have it fixed by tonight!

**Comment by stefanwagnerdev** [comment]
same problem for me

**Comment by yocontra** [comment]
I thought I had this fixed... bompo can you post your app logs?

**Comment by stefanwagnerdev** [comment]
no logs :/

``` bash
 bompo@linux-desktop:~$ nodester app start helloworldbompo
 The "sys" module is now called "util". It should have a similar interface.
 nodester info Attemping to start app: helloworldbompo
 nodester warn false
```

``` bash
bompo@linux-desktop:~$ nodester app logs helloworldbompo
The "sys" module is now called "util". It should have a similar interface.
nodester info { response: '{"status":"failure","message":"No logs available."}\n',
  errorCode: 500 }
nodester ERROR Fatal Error! API Response: {"status":"failure","message":"No logs available."}

Reason: No logs available.
```

**Comment by rnrodrigues** [comment]
Same problem for me too:

$ nodester app info twoclouds
nodester info Gathering information about: twoclouds
nodester info twoclouds on port 12XXX running: failed-to-start

$ nodester app start twoclouds
nodester info Attemping to start app: twoclouds
nodester warn false

**Comment by andy-novikov** [comment]
same here, fail to start hello world

**Comment by borisoks** [comment]
same here, same output as rnrodrigues

also http://irc.nodester.com/ is non-responsive

**Comment by newtriks** [comment]
I am having the same problem, no logs to show also as the app failed-to-start?

**Comment by flukeout** [comment]
I am having the same problem. I can't start the basic hello world app and it's stuck in failed-to-start status. No logs either.

**Comment by yocontra** [comment]
Discussion of this issue has moved to https://github.com/nodester/nodester/issues/241

**Comment by VersatileCapitalist** [comment]
I cannot start my nodester app.Its stuck in failed-to-start status and no logs either. I received my coupon yesterday. Any ideas?

**Comment by chrismatthieu** [comment]
Things to check:

Are you using the port that Nodester assigned your app? nodester app info
appname
Did you add your npm modules to your app and restart it?
Is your RSA properly set?
Is nodester app logs appname giving you any hints?

Check out our FAQs at http://nodester.com/help.html#faq

Regards,
Chris

On Mon, Jan 16, 2012 at 3:09 PM, VersatileCapitalist <
reply@reply.github.com

**Comment by VersatileCapitalist** [comment]
Hey Chris, Fantastic turnaround!

Should have mentioned this before but the error is for hello world example app. I just followed the steps to create a new test app from the welcome email. 

I belive my RSA is properly set. I am able to perform various operations (e.g. nodester create app). git is installed locally.

Some more info - 

C:\Users\atomar>nodester app restart iwitness
The "sys" module is now called "util". It should have a similar interface.
nodester info Attemping to restart app: iwitness
nodester warn false

C:\Users\atomar>nodester app logs iwitness
The "sys" module is now called "util". It should have a similar interface.
nodester info { response: '{"status":"failure","message":"No logs available."}\n',
  errorCode: 500 }
nodester ERROR Fatal Error! API Response: {"status":"failure","message":"No logs available."}

Reason: No logs available.

node -v
v0.6.7

npm -v
1.1.0-beta-10

nodester -v
The "sys" module is now called "util". It should have a similar interface.
0.2.35

Windows Vista

Many Thanks!
Ashish

**Comment by VersatileCapitalist** [comment]
hmm..maybe i spoke too soon....
$ git clone git@nodester.com:/node/git/atomar/5606-75eee94c4479d07b41e0aa6132ac029f.git iwitness2
Cloning into 'iwitness2'...
Nodester!
Connection closed by 50.16.203.53
fatal: The remote end hung up unexpectedly
